packages/devtools_app/lib/src/screens/profiler/cpu_profile_model.dart

```diff
@@ -712,8 +718,8 @@ class CpuStackFrame extends TreeNode<CpuStackFrame>
     String? parentId,
     int? sourceLine,
     CpuProfileMetaData? profileMetaData,
-    bool copySampleCountsAndTags = true,
-    bool resetInclusiveSampleCount = true,
+    bool copySampleCounts = true,
+    bool resetInclusiveSampleCounts = true,
```

Comment:
nit: rename back to `resetInclusiveSampleCount` since it is a singular value `CpuStackFrame.inclusiveSampleCount`